modules/NodeColorManager/src/main/nbm/manifest.mf

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+OpenIDE-Module-Name: Node Color Manager
+OpenIDE-Module-Short-Description: allow you share node colors among multi graph
+OpenIDE-Module-Long-Description: \
```

Comment:
There's something wrong with the formatting of this line as Java refuses to compile this manifest. You should probably use HTML tags like `Useful if you want to keep the same node to be the same color across diffrent graph/projects, by using a file to keep track of node colors among multi-graph. Note that the tool identify nodes by their labels, so your graph must have unique label for each node. If two or more node has same label, only one will be restored. When restore from file, only the nodes which match the labels in file would be colored, others will remain their original color. <br/>
    source code: <a href="https://github.com/songjinghe/gephi-plugins-1">https://github.com/songjinghe/gephi-plugins-1</a><br/>
    more details: <a href="https://github.com/songjinghe/gephi-plugins-1/wiki">https://github.com/songjinghe/gephi-plugins-1/wiki</a>`